cves/kernel/CVE-2022-28389.yml

```diff
@@ -75,23 +98,19 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs:
+  - https://bugzilla.redhat.com/show_bug.cgi?id=2073086
+  - https://bugzilla.redhat.com/show_bug.cgi?id=2073087
```

Comment:
I do not think either of these bugzilla references are relevant as the instructions want kernel bugzillas